jsapp/js/components/common/koboSelect.scss

```diff
@@ -100,6 +100,15 @@ $k-select-menu-padding: sizes.$x6;
   margin: 0;
 }
 
+.k-select__error {
+  font-size: sizes.$x12;
+  line-height: 1.6;
+  font-weight: 400;
+  font-style: normal;
+  color: colors.$kobo-red;
+  margin-top: sizes.$x6;
+}
+
```

Comment:
Could this message get positioned somehow so that the layout doesn't jump when it appears/disappears? Also, it would be nice if the error state had the same red border as the `TextBox`.

Reply:
The layout jumping is actually consistent with how the TextBox error is being displayed. I think this will require us having similar errors in all the common components, and then unfiying the functionality.

Reply:
I added the styles for the select trigger (for all the types: outline, blue, and gray)

Reply:
I agree that it's consistent with how the `TextBox` works, but I wonder if keeping it consistent is worth it in this case - especially since the refactor of the of the `KoboSelect` fields makes them look more distinct from the `TextBox`es. You could keep the same layout/position and avoid the content jumping by adding a very short animation to a height property/transform of the error message.

Still, I take your point - this shouldn't hold up this task👍I'll mark it as resolved.